extractors/python_functions/work_of_art_extraction/code_snippet.md

```diff
@@ -1,9 +1,8 @@
 ```python
-from typing import List, Tuple
 YOUR_ATTRIBUTE: str = "text" # only text attributes
 YOUR_LABEL: str = "work of art"
 
-def work_of_art_extraction(record) -> List[Tuple[str, int, int]]:
+def work_of_art_extraction(record):
```

Comment:
extractor

Reply:
See other comments about the extraction naming.